lib/papertrail/http_client.rb

```diff
@@ -42,47 +42,27 @@ def get(path, params = {})
       if params.size > 0
         path = "#{path}?#{build_nested_query(params)}"
       end
-      attempts = 0
-      begin
+
+      attempt_http tries: 3, wait: 5.0 do
```

Comment:
I just realized this instance of `#attempt_http` is using the same number of tries as all the others. I think it's fine to hard code 3 tries in the method and wait to expose that option in the future if needed.